hackathon_site/dashboard/frontend/package.json

```diff
@@ -27,13 +26,17 @@
     "redux": "^4.0.5",
     "redux-mock-store": "^1.5.4",
     "redux-thunk": "^2.3.0",
+    "sass": "^1.32.12",
     "yup": "^0.29.1"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jest-environment-jsdom-sixteen",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "i": "yarn install --frozen-lockfile",
+    "pretty": "yarn prettier --write 'src/**/*.(scss|js)'",
+    "precheck": "yarn pretty && yarn test -o -u --watchAll=false"
```

Comment:
once I add Flow to this, precheck will just be something you should run before you commit